.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+# Description
+
+Summary of changes
+
+* ... 
+* ...
+## Resolved Issues
```

Comment:
Should we have whitespace before?
```suggestion

## Resolved Issues
```

Reply:
Also, is there a reason why some of the bullet points have * and others -?

Reply:
They are interchangeable